pkgs/by-name/sf/SFCGAL/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  cmake,
+  cgal,
+  boost,
+  mpfr,
+  gmp,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sfcgal";
+  version = "1.5.2";
+
+  src = fetchurl {
```

Comment:
Have you tried fetchFromGitLab?

Reply:
I have not. Thank you, I don't know why I didn't check if that was a thing.